lib/ClangImporter/ClangImporter.cpp

```diff
@@ -71,11 +71,13 @@
 #include "llvm/Support/FileSystem.h"
 #include "llvm/Support/Memory.h"
 #include "llvm/Support/Path.h"
+#include "llvm/Support/VirtualFileSystem.h"
 #include "llvm/Support/YAMLParser.h"
 #include "llvm/Support/YAMLTraits.h"
-#include "llvm/Support/VirtualFileSystem.h"
 #include <algorithm>
+#include <string>
 #include <memory>
+#include <string>
```

Comment:
I assume the double inclusion of the string header is a mistake?

Reply:
This is from a different PR  (which Im branched off of) but this is a good catch thanks!

Reply:
🙏

Reply:
Addressing in this PR actually since this is a follow up and cleaning up this include is fine here I think